akka-remote/src/main/scala/akka/remote/artery/compress/InboundCompressions.scala

```diff
@@ -114,7 +114,7 @@ private[remote] abstract class InboundCompression[T >: Null](
   private[this] val state: AtomicReference[InboundCompression.State[T]] = new AtomicReference(InboundCompression.State.empty)
 
   // TODO calibrate properly (h/w have direct relation to preciseness and max capacity)
-  private[this] val cms = new CountMinSketch(100, 100, System.currentTimeMillis().toInt)
+  private[this] val cms = new CountMinSketch(16, 1024, System.currentTimeMillis().toInt)
```

Comment:
I don't think so. Moreover it's should be configuration parameter and depends on traffic you send. In some cases CMS with smaller width would perform better and have accepted estimation error level.